File: All Codes/URI_1021.java
import java.util.Scanner;

/**
 * URI_1021
 */
public class URI_1021 {

    public static void main(String[] args) {
        try (Scanner input = new Scanner(System.in)) {
            int arr[] = {10000, 5000, 2000, 1000, 500, 200, 100, 50, 25, 10, 5, 1};
            String title = "nota(s) de R$";
            int temp, n;
            double N = input.nextDouble();

            n = (int)(N * 100 + 0.00001);
            System.out.printf("NOTAS:\n");

            for (int i = 0; i < arr.length; i++) {
                temp = n / arr[i];
                n %= arr[i];
                if (i == 6) {
                    title = "moeda(s) de R$";
                    System.out.printf("MOEDAS:\n");
                }
                System.out.printf("%d %s %.2f\n", temp, title, arr[i]/100.00);
            }

        }
    }
}

File: All Codes/URI_1021.py
a = float(input())
b = [100.00, 50.00, 20.00, 10.00, 5.00, 2.00, 1.00, 0.50, 0.25, 0.10, 0.05, 0.01]
c = "nota(s) de R$"

print(f"NOTAS:")
for x in b:
    print(f"{int(a / x)} {c} {x:.2f}")
    a %= x
    a = float(f"{a:.2f}") # I love f-string..Ummahh :* 
    if x == 2.00:
        print(f"MOEDAS:")
        c = "moeda(s) de R$"